docs/CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Antares-Xpansion changelog
 =================
 
+v1.0.2 (01/2023)
```

Comment:
Can we add bullet points before items to uniformize with the rest of the docs and mention that now we zip the entire output and not just the MPS ?

Reply:
We will do this in the develop branch finally